x-pack/docs/en/watcher/actions/email.asciidoc

```diff
@@ -379,14 +379,18 @@ http://aws.amazon.com/ses[Amazon Simple Email Service] (SES) SMTP service:
 --------------------------------------------------
 xpack.notification.email.account:
     ses_account:
+        email_defaults:
+            from: <email address of service account> <1>
         smtp:
             auth: true
             starttls.enable: true
             starttls.required: true
-            host: email-smtp.us-east-1.amazonaws.com <1>
+            host: email-smtp.us-east-1.amazonaws.com <2>
             port: 587
             user: <username>
 --------------------------------------------------
+<1> Some organizations configure Amazon SES to validate that the `from` field is a
+    valid local email account.
```

Comment:
What would you think about rephrasing this in a way that is inline with the comment below. For example, (this is just a suggestion):

```
<1> in certain cases `email_defaults.from` is validated by Amazon SES to ensure that it is a valid local email account.
```